extension/src/config/config.js

```diff
@@ -6,7 +6,8 @@ function getEnvConfig () {
     logLevel: process.env.LOG_LEVEL,
     apiExtensionBaseUrl: process.env.API_EXTENSION_BASE_URL,
     keepAliveTimeout: !Number.isNaN(process.env.KEEP_ALIVE_TIMEOUT) ? parseFloat(process.env.KEEP_ALIVE_TIMEOUT, 10)
-      : undefined
+      : undefined,
+    disableEnsureResources: process.env.DISABLE_ENSURE_RESOURCES === 'true' || false
```

Comment:
Why not use here same naming as the env name itself?

Reply:
I'm not sure what you mean, as far as I can see it is the same, just camelcase.